tools/rack/rack.plugin/META-INF/MANIFEST.MF

```diff
@@ -91,7 +91,7 @@ Bundle-ClassPath: .,
  lib/opencsv.jar,
  lib/sparqlGraphLibrary.jar,
  lib/viz.js-graphviz-java.jar
-Bundle-RequiredExecutionEnvironment: JavaSE-11
+Bundle-RequiredExecutionEnvironment: JavaSE-17
```

Comment:
If we update this file to Java 17, we should also update `tools/rack/rack.test/META-INF/MANIFEST.MF` too.